src/docs/product/alerts/alert-types.mdx

```diff
@@ -18,7 +18,7 @@ You can create two types of alerts:
 
 ## Issue Alerts
 
-Issue alerts trigger whenever any issue in a project matches the specified criteria. These criteria might be, for example, a resolved issue re-appearing or an issue affecting many users.
+Issue alerts trigger whenever a new event is received and any issue in a project matches the specified criteria. These criteria might be, for example, a resolved issue re-appearing or an issue affecting many users.
```

Comment:
It almost sounds like the two states are disconnected, so I'm trying to create some connective tissue between them. Does this work?

```suggestion
Issue alerts trigger whenever a new event is received for any issue in a project matching the specified criteria. These criteria might be, for example, a resolved issue re-appearing or an issue affecting many users.
```